electron/browser/view-manager.ts

```
@@ -0,0 +1,82 @@
+    // Add views back in order. addChildView brings the added view to the top
+    // relative to its siblings managed by this parent.
+    // Adding lowest zIndex first means highest zIndex will end up visually on top.
+    sortedViews.forEach(([view]) => {
+      try {
+        // Check if view is still valid (not destroyed, etc.) - View API has no direct 'isDestroyed'
+        // Rely on addChildView potentially throwing an error if the view is invalid
+        this.parentView.addChildView(view);
+      } catch (error) {
+        console.error(`Failed to add/reorder view during reorder:`, error);
+        // Attempt to find the ID for the failed view to remove it from the map
+        const failedViewEntry = Array.from(this.views.entries()).find(([, zIndex]) => zIndex === zIndex);
```

Biome (1.9.4)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)